sketchy/controllers/tasks.py

```diff
@@ -80,9 +81,15 @@ def do_capture(status_code, the_record, base_url, model='capture'):
     """
     # Make sure the the_record
     db.session.add(the_record)
+    
+    # Generate Unix Timestamp of capture date and append to the capture_name, to guarantee uniqueness.
+    from datetime import datetime
```

Comment:
Can you move this to the top of the file?  I don't want to import datetime every time this function is called.